webapp/components/Notification/Notification.spec.js

```diff
@@ -198,4 +199,71 @@ describe('Notification', () => {
       })
     })
   })
+
+  describe('given a notification about the user has filed a report', () => {
+    beforeEach(() => {
+      propsData.notification = {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T19:13:50Z" title="Thursday, January 23rd 2020, 8:13:50 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -198,4 +199,71 @@ describe('Notification', () => {
       })
     })
   })
+
+  describe('given a notification about the user has filed a report', () => {
+    beforeEach(() => {
+      propsData.notification = {
+        reason: 'filed_report_on_resource',
+        from: {
+          __typename: 'Report',
+          id: 'reportOnUser',
+          filed: [
+            {
+              reasonCategory: 'discrimination_etc',
+              reasonDescription: 'This user is harassing me with bigoted remarks!',
+              reportedResource: {
+                __typename: 'User',
+                id: 'badWomen',
+                slug: 'mrs.-badwomen',
+                name: 'Mrs. Badwomen',
+              },
+            },
+          ],
+        },
+      }
+    })
+
+    it('renders reason', () => {
+      wrapper = Wrapper()
+      expect(wrapper.find('.reason-text-for-test').text()).toEqual(
```


same as :point_up: ... if you added this class, please use `data-test="reason-text"` as we decided we don't want classes for testing purposes.
If a class exists and can be used to select it, we should use it, otherwise, we should add `data-test`
If you didn't add this, could you please refactor it?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T10:23:00Z" title="Wednesday, February 5th 2020, 11:23:00 am +01:00">Feb 5, 2020</time>_

---


Done dear 🤗